reference/composedshader.md

```diff
@@ -0,0 +1,105 @@
+## ComposedShader
+
+```
+ComposedShader {
+  vrmlField MFNode fields []
+  vrmlField MFNode parts  []
```

Comment:
In my point of view, the priority rules to choose names are the following:

1. Follow X3D
2. Have descriptive names for our own
3. Follow X3DOM (if good enough)